examples/network_functions/go-gtp/pgw/pgw_default.yml

```diff
@@ -1,6 +1,5 @@
 local_addresses:
   s5c_ip: "172.25.1.14"
   s5u_ip: "172.25.0.14"
-  sgi_ip: "10.0.1.254"
```

Comment:
sgi_ip is still included [here](https://github.com/wmnsk/go-gtp/blob/master/examples/gw-tester/pgw/pgw.yml). Is there a reason to remove it?